recipes/parserutils/meta.yaml

```
@@ -0,0 +1,46 @@
+{% set name = "parserutils" %}
+{% set version = "2.0.1" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/parserutils-{{ version }}.tar.gz
+  sha256: f927e69779d81db508db98e6e8ec331f90ff31ff2868161d1cce30fffe92bec3
+
+build:
+  number: 0
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  host:
+    - pip
+    - python >=3.6,<4.0
+  build:
+    - poetry
```

Try making a `host` dependency?
```suggestion
    - python >=3.6,<4.0
    - poetry
```

I mean, I don't expect that to change anything (since I see it in the log being installed in the environment), but that's how it should be in the end anyway.